apps/ui-kit/src/lib/components/organisms/dialog/Dialog.tsx

```diff
@@ -68,8 +68,8 @@ const DialogContent = React.forwardRef<
         }, [containerId]);
         const positionClass =
             position === DialogPosition.Right
-                ? 'right-0 h-screen top-0 w-full'
-                : 'max-h-[60vh] left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2 rounded-xl';
+                ? 'overflow-hidden right-0 h-screen top-0 w-full'
```

Comment:
this is a bit tricky :see_no_evil:  the overflow is good now, but the title disappears
![Screenshot from 2025-03-14 08-29-38](https://github.com/user-attachments/assets/0ed24d68-454f-4f33-9cc6-5e0e899fa494)